python/lsst/daf/butler/registries/sqlRegistry.py

```diff
@@ -117,15 +117,28 @@ def registerDatasetType(self, datasetType):
 
         Raises
         ------
-        KeyError
-            Dataset is already registered.
         ValueError
-            DatasetType is not valid for this registry.
+            DatasetType is not valid for this registry or is already registered
+            but not identical.
         """
+        # Check if datasetType is valid for this Registry
```

Comment:
This comment isn't useful as-is; it just repeats what the code below says.  But it might be useful to have a comment that says what "valid for this Registry" means (or at least mentions a way a DatasetType may not be valid).

Reply:
True, it was more of a remnant of a TODO.